storage.js

```diff
@@ -0,0 +1,3 @@
+import {AsyncLocalStorage} from 'async_hooks';
+
+export const storage = new AsyncLocalStorage();
```

Comment:
you want a single instance of `AsyncLocalStorage` for the whole App. See it as the equivalent of a threadLocal

Reply:
yes i understand, so main goal, to have ALS some where in singleton and re-use it/reimport inside the project

Reply:
i've expected more convenient API here

Reply:
It is expected that multiple people to have tehir own instance. We want to enable them to hide it from each others.

Things like CLS-like API could be done on npm package side but we would like to avoid it in core

Reply:
you mean like to have in certain `ALS.run` scope, have another `ALS.run` scope and don't share storage inside child scope?

Reply:
I mean more than 1 call to `new AsyncLocalStorage()` within the same app owned by different entities.